edx_proctoring/api.py

```diff
@@ -1054,6 +1054,23 @@ def get_filtered_exam_attempts(course_id, search_by):
     return [ProctoredExamStudentAttemptSerializer(active_exam).data for active_exam in exam_attempts]
 
 
+def get_course_completion_date(course_id, username):
+    """
+    return course completion date if course is completed otherwise None
+    Note:
+
+    1) If all the proctored exams have completion date then course is complete.
+    2) Course completion date will be the last proctored exam's completion date
+    """
+    exam_attempts = ProctoredExamStudentAttempt.objects.get_proctored_exam_attempts(course_id, username)
+    for attempt in exam_attempts:
```

Comment:
Can we also find the number of Proctored exams in the course in addition to the attempts, just for completion?